src/store.c

```diff
@@ -339,47 +407,104 @@ static void writer_close(
     close(store->fd);
 }
 
+static struct rill_pairs* flip_values(struct rill_pairs* pairs)
```

Comment:
I would add this function to `rill_pairs` as it deals exclusively with the rill pairs data.